public/js/ajax.js

```diff
@@ -0,0 +1,25 @@
+
+var ajaxForm = document.getElementById("contactForm")
+ajaxForm.addEventListener("submit", function(e){
+    e.preventDefault();
+    var form = e.target;
+    
+    //make a new object which grabs the form fields value and then save them to file using ajax
+    var data= new Object();
+    let title1=document.querySelector("#title").value
```

Comment:
You can assign the variable right here:

```
data.title = document.querySelector("#title").value
```

This saves lines of code an unnecessary memory assignment.  The more concise you can be, the better :)